contrib/!lang/html/packages.el

```diff
@@ -197,10 +203,21 @@
   (use-package slim-mode
     :defer t))
 
+(defun html/init-jade-mode ()
+  (use-package jade-mode
+    :defer t
+    :config (add-hook 'jade-mode-hook #'highlight-indentation-mode)))
```

Comment:
PS we should enable it in layers that use indentation (python, ruby, sass, slim) etc.